integreat_cms/textlab_api/textlab_api_client.py

```diff
@@ -43,15 +43,23 @@ def login(self) -> None:
         response = self.post_request("/user/login", data)
         self.token = response["token"]
 
-    def benchmark(self, text: str) -> float | None:
+    def benchmark(self, text: str, text_type: int = 420) -> float | None:
```

Comment:
As I understand the API, we currently have no need to change the ID (ever again?).

Reply:
But it is true that anyone setting up integreat-cms on their own servers would definitely need to, since 420–424 were configured specifically for us. Meaning that any other textlab account likely won't get access to it. I think.